app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
- [ ] [OPTIONAL] It would  be beneficial if you had included a sign out button to help in testing how the app works for various users in the system. You can use the code below for that:

```
<li class="nav-item">
  <%= link_to "Sign out", destroy_user_session_path, class: 'nav-link', data: { turbo_method: :delete } %>
</li>
```